common/containers/Tabs/GenerateWallet/components/WalletTypes.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import translate from 'translations';
+import { WalletType } from '../GenerateWallet';
+import { NewTabLink } from 'components/ui';
+import './WalletTypes.scss';
+
+interface Props {
+  onSelect(type: WalletType): void;
+}
+
+export default class WalletTypes extends React.Component<Props, {}> {
```

Comment:
This could be an SFC too

Reply:
Would be dope if there was a linter rule or something that caught implementations of React.Component that only had a render method. I'll convert this, but lmk if you think that's out there.